### ISSUE TITLE
Conflict with WordPress SEO plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#189]
```
What functionality do you want to see?

The CCTM plugin appears to be incompatible with the WordPress SEO plugin ( 
http://wordpress.org/extend/plugins/wordpress-seo/ )

To repeat the behavior, first deactivate the CCTM plugin, then:

1. Install the WordPress SEO plugin
2. Use its dashboard to set it up
3. Generate an XML sitemap: http://yourstie.com/sitemap_index.xml  (the page 
should show a list of posts and pages).
4. Activate the CCTM plugin and refresh the sitemap page -- it gets a 404 error.


```

Original issue reported on code.google.com by `fireproofsocks` on 2 Oct 2011 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
There seems to be a caching-type of bug with the WordPress SEO plugin -- I've 
filed a bug there: 
http://yoast.com/bugs/wordpress-seo/tickets/xml-sitemap-rules-do-not-reset-in-an
-expected-way/

This definitely has something to do with the flushing of permalink rules.  If 
you open up the includes/CCTM.php and you comment out the following 2 lines:

        global $wp_rewrite;
        $wp_rewrite->flush_rules();

Then the WP SEO XML site map page works.  But why does this built-in WP 
function conflict with the SEO plugin?  
```

Original comment by `fireproofsocks` on 2 Oct 2011 at 11:02

**Comment by GoogleCodeExporter** [comment]
```
Interesting: the priority of the action will change the behavior here.

Edit the includes/loader.php file.  Change the following line:

    add_action('init', 'CCTM::register_custom_post_types', 1 );

So it uses a priority of something GREATER than the default 10, e.g.:

    add_action('init', 'CCTM::register_custom_post_types', 11 );

That fixes this particular problem.  I don't honestly know what affect the 
priority has on other plugins etc... maybe it's better for the CCTM to register 
post-types after other plugins instead of before.  

This fix will be included in 0.9.4.
```

Original comment by `fireproofsocks` on 2 Oct 2011 at 11:10
- Changed state: **Fixed**
